inst/report/KRIReportBySite.Rmd

```diff
@@ -106,8 +107,18 @@ title: "`r study_id` Assessment Overview"
 
 
 ## metadata report if longitudinal data is provided
-if (exists('status_study', where = params) && !is.null(params$status_study)) {
+
+status_study_exists <- exists('status_study', where = params) && !is.null(params$status_study)
+
+if (status_study_exists) {
+  
+  # Make Study Status table
   MakeStudyStatusTable(params$status_study)
+  
+  # pull number of actual subjects
+  subjects <- purrr::map(assessment, ~length(.$lData$dfSUBJ[[Read_Mapping()$dfSUBJ$strIDCol]])) %>%
+   unlist() %>%
+   max()
```

Comment:
I can't figure out why the solution I suggested isn't working 🤷so I think this is great for now, but we'll want to figure out how to pull the mapping based on the `lInput$lMapping` object in the future.